chart/openfaas/values.yaml

```diff
@@ -116,6 +116,16 @@ ingress:
   tls:
     # Secrets must be manually created in the namespace.
 
+ingressOperator:
+  image: openfaas/ingress-operator:0.4.0
+  replicas: 1
+  create: false
+  resources:
+    limits:
+      memory: 128Mi
```

Comment:
As a rule we don't set hard limits in the helm chart and leave that up to the user to decide. The requests values are set by us though.

Reply:
Ok, fixed!